ddl/schema.go

```diff
@@ -128,7 +128,11 @@ func onDropSchema(t *meta.Meta, job *model.Job) (ver int64, _ error) {
 func getIDs(tables []*model.TableInfo) []int64 {
 	ids := make([]int64, 0, len(tables))
 	for _, t := range tables {
-		ids = append(ids, t.ID)
+		if t.GetPartitionInfo() == nil {
```

Comment:
The partitioned table itself should included. @crazycs520 @ciscoxll 

```
ids = append(ids, t.ID)
if t.GetPartitionInfo() != nil {
    ids = append(ids, getPartitionIDs(t)...)
}
```

Reply:
done.